test/x86/avx.c

```diff
@@ -9785,6 +9791,25 @@ test_simde_mm_maskload_pd (SIMDE_MUNIT_TEST_ARGS) {
   return 0;
 }
 
+#if !defined(HEDLEY_MSVC_VERSION)
+static int
+test_simde_mm_maskload_pd_no_illegal_memory_access (SIMDE_MUNIT_TEST_ARGS) {
```

Comment:
New test functions need to be added to the list at the end of the file. Make sure to also surround those entries with `!defined(HEDLEY_MSVC_VERSION)`

https://github.com/simd-everywhere/simde/blob/28246b76b11331f9f235923175accffe90f47546/test/x86/avx.c#L16241